test/suite/test_backup15.py

```diff
@@ -293,7 +258,8 @@ def test_backup15(self):
             else:
                 self.take_incr_backup()
                 self.take_full_backup()
-            self.compare_backups(self.uri)
+
```

Comment:
I find that the `compare` call next to the `take_backup` calls is clear and shows it is related to taking the backups. I would suggest you don't add this blank line.

Reply:
yep! removed.